internal/backends/winit/renderer/skia/itemrenderer.rs

```diff
@@ -647,10 +647,10 @@ impl<'a> ItemRenderer for SkiaRenderer<'a> {
         from_skia_rect(&self.canvas.local_clip_bounds().unwrap_or_default()) / self.scale_factor
     }
 
-    fn translate(&mut self, x: LogicalLength, y: LogicalLength) {
+    fn translate(&mut self, distance: LogicalVector) {
         self.canvas.translate(skia_safe::Vector::from((
-            (x * self.scale_factor).get(),
-            (y * self.scale_factor).get(),
+            (distance.x_length() * self.scale_factor).get(),
```

Comment:
You can multiply the vector by the scale factor then access the X and y directly

Reply:
Good idea